tests/com/twitter/intellij/pants/util/PantsUtilTest.java

```diff
@@ -24,19 +27,29 @@ public void testIsPantsProjectFile() {
   }
 
   public void testFindJdk() {
-    Optional<File> executable = PantsUtil.findPantsExecutable(getProjectFolder());
-    assertTrue(executable.isPresent());
-    Optional<Sdk> sdkA = PantsUtil.getDefaultJavaSdk(executable.get().getPath());
-    assertTrue(sdkA.isPresent());
-    ApplicationManager.getApplication().runWriteAction(new Runnable() {
-      @Override
-      public void run() {
-        ProjectJdkTable.getInstance().addJdk(sdkA.get());
-      }
+    final File executable = PantsUtil.findPantsExecutable(getProjectFolder()).get();
+    final Sdk sdkA = getDefaultJavaSdk(executable.getPath()).get();
+    List<Sdk> sdkResults = Arrays.stream(ProjectJdkTable.getInstance().getAllJdks())
+      .filter(jdk -> jdk == sdkA)
+      .collect(Collectors.toList());
+    assertTrue(sdkResults.size() == 1);
```

Comment:
`assertEqual` same for below

Reply:
Just pushed a commit doing this.

Reply:
Thanks, pushed a commit which makes the changes.